pages/mesosphere/dcos/services/hdfs/2.8.0-3.2.1/release-notes/index.md

```diff
@@ -8,6 +8,15 @@ model: /mesosphere/dcos/services/hdfs/data.yml
 render: mustache
 ---
 
+# Release Notes for HDFS version 2.8.0-3.2.1
+
+## Updates
+- Updated SDK to version `0.57.3`. For more information see release-notes for previous SDK releases:
+  - [0.57.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.3)
```

Comment:
I think we can provide links to all in-between SDK release , also we have already mentioned till 0.57.0 in previous version 2.7.0-3.2.1, so we can remove that and provide link to 0.57.1 ,0.57.2 and 0.57.3

Reply:
Updated.